branded_desktop_client/update_branded_desktop_clients.rst

```diff
@@ -2,6 +2,40 @@
 Updating Your Branded Desktop Clients
 =====================================
 
+Branded desktop clients can use the same auto update feature as the regular
+ownCloud client. (Preventing Automatic Updates ==> https://doc.owncloud.com/desktop/latest/autoupdate.html#preventing-automatic-updates)
+
+Using updates.owncloud.com
+--------------------------
+
+The easy way is, to notify your ownCloud support about the location where
+you uploaded your new generated branded client binaries.
+
+You need to perform the following steps:
+
+#. Generate branded clients.
+#. Upload branded clients to your Web server.
+
+   -  Windows example:
```

Comment:
@michaelstingl I don't really understand what is meant by these examples. Can you clarify please?